src/Types/Input/EnumInputType.hack

```diff
@@ -25,10 +25,7 @@ abstract class EnumInputType extends NamedInputType {
     }
 
     final protected function assertValidVariableValue(mixed $value): this::TCoerced {
```

Comment:
You can actually delete this entirely and make the parent class's implementation final, I think.